### DISCUSSION TITLE
Fastlane.swift UI helper methods not available

### DISCUSSION THREAD

**Discussion by vexonius** [fastlane/fastlane#20383] [Q&A]
Hi, I'm rewriting a fastlane script in swift and would like to improve and extend console experience for our app build process and would like to use UI.selection helper method that is only available in Ruby... Is there a way to access it from Swift environment?

Something like this:
`project = UI.select("Select your project: ", ["Test Project", "Test Workspace"])
`

Thanks

**Comment by HarshilShah** [comment]
Bumping this, would be really nice to have this working in Fastlane.swift
